libs/components/layout/src/lib/modules/box/box-header.component.scss

```diff
@@ -1,9 +1,21 @@
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides('sky-box') {
+  --sky-override-box-header-padding-right: var(--sky-margin-inline-xl);
+}
+
+@include compatMixins.sky-modern-overrides('sky-box') {
+  --sky-override-box-header-padding-right: var(--sky-margin-inline-xl);
+}
+
 sky-box-header {
   flex: 1 0;
-  order: 0;
```

Comment:
order wasn't doing anything here--they're already (and will always be) in that order